configs/data/the_well.yaml

```
@@ -19,6 +19,7 @@ max_dt_stride: 1
 
 # DataLoader configuration
 batch_size: 16
+num_workers: 8  # Explicitly set to avoid OOM from too many workers
```

Setting to 0 appears to be only compatible option that does not use all RAM at the moment:
```suggestion
num_workers: 0
```